h2o-py/tests/testdir_algos/xgboost/pyunit_xgboost_eval_metric_stopping.py

```diff
@@ -1,9 +1,15 @@
 import sys
 
 sys.path.insert(1, "../../../")
+import h2o
```

Comment:
Importing only the upload_custom_metric function would be better than importing the whole h2o package.

Reply:
done, thanks